.github/workflows/readme.yml

```diff
@@ -1,17 +0,0 @@
-name: 'Tests: readme'
```

Comment:
We might not need a separate workflow for this as the doc generation is just a `lint` script in package.json now.

Reply:
I'm not a fan of implicit magic script name prefix conventions, but it should automatically run without a separate workflow as part of linting.